images/management-api/bin/gravitee

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
```

Comment:
Why providing this binary instead of using the one from the container?

Reply:
Hi,
i tried to make less changes as possible respect to the initial master so due the fact in the original binary there is the following code:

```
if [ -f "/etc/gravitee-gateway/gravitee.yml" ]
then
        GRAVITEE_OPTS="-Dgravitee.conf=/etc/gravitee-gateway/gravitee.yml"
fi
```
the configuration supplied from outside the container wasn't read

So, I had to change as follow:

```
if [ -f "$GRAVITEEIO_HOME/config/gravitee.yml" ]
then
	GRAVITEE_OPTS="-Dgravitee.conf=$GRAVITEEIO_HOME/config/gravitee.yml"
fi
```